Upgrade Lodash to version 4.17.21

Our current version of the `easygraphql-tester` package is at v5.1.6. It requires a vulnerable version of lodash.

We need to fix this vulnerability so that both packages are upgraded appropriately.

![image](https://user-images.githubusercontent.com/16875803/167870731-c89394fb-a92b-4278-8d30-c11f5a5d2df3.png)

We are getting the following messages when trying to deploy the code on the Heroku server. 

```

-----> Building on the Heroku-20 stack
-----> Using buildpack: heroku/nodejs
-----> Node.js app detected
       
-----> Creating runtime environment
       
       NPM_CONFIG_LOGLEVEL=error
       NODE_VERBOSE=false
       NODE_ENV=production
       NODE_MODULES_CACHE=true
       
-----> Installing binaries
       engines.node (package.json):  unspecified
       engines.npm (package.json):   unspecified (use default)
       
       Resolving node version 16.x...
       Downloading and installing node 16.17.0...
       Using default npm version: 8.15.0
       
-----> Restoring cache
       - npm cache
       
-----> Installing dependencies
       Installing node modules
       npm ERR! code EUSAGE
       npm ERR! 
       npm ERR! `npm ci` can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.
       npm ERR! 
       npm ERR! Invalid: lock file's inquirer@8.2.0 does not satisfy inquirer@8.2.4
       npm ERR! Invalid: lock file's marked@4.0.10 does not satisfy marked@2.1.3
       npm ERR! Missing: typescript@4.8.2 from lock file
       npm ERR! 
       npm ERR! Clean install a project
       npm ERR! 
       npm ERR! Usage:
       npm ERR! npm ci
       npm ERR! 
       npm ERR! Options:
       npm ERR! [--no-audit] [--foreground-scripts] [--ignore-scripts]
       npm ERR! [--script-shell <script-shell>]
       npm ERR! 
       npm ERR! aliases: clean-install, ic, install-clean, isntall-clean
       npm ERR! 
       npm ERR! Run "npm help ci" for more info
       
       npm ERR! A complete log of this run can be found in:
       npm ERR!     /tmp/npmcache.3IHsW/_logs/2022-08-28T15_55_01_181Z-debug-0.log
-----> Build failed
       
       We're sorry this build is failing! You can troubleshoot common issues here:
       https://devcenter.heroku.com/articles/troubleshooting-node-deploys
       
       Some possible problems:
       
       - Node version not specified in package.json
         https://devcenter.heroku.com/articles/nodejs-support#specifying-a-node-js-version
       
       Love,
       Heroku
       
 !     Push rejected, failed to compile Node.js app.
 !     Push failed

```

--- a/package.json
+++ b/package.json
@@ -83,7 +83,6 @@
     "yarn": "^1.22.10"
   },
   "devDependencies": {
-    "easygraphql-tester": "^5.1.6",
     "eslint": "^7.29.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-config-prettier": "^8.3.0",

